zerver/openapi/zulip.yaml

```diff
@@ -5271,6 +5271,55 @@ paths:
                         "result": "error",
                         "msg": "Draft does not exist",
                       }
+  /reminders:
+    post:
+      operationId: create-reminder
+      tags: ["reminders"]
+      summary: Create a reminder
+      description: |
+        Create a new [reminder](/help/schedule-a-reminder).
+
+        New in Zulip 8.0 (feature level 216).
+      parameters:
+        - name: message_id
+          in: query
+          description: |
+            The ID of the message to be sent when the reminder is due.
```

Comment:
This is an existing message, right? In that case it's not an ID of a message to be sent.

"of the message to be reminded of"?

Reply:
Done.